core/src/main/resources/error/error-classes.json

```diff
@@ -1279,6 +1279,11 @@
           "The ANALYZE TABLE FOR COLUMNS command can operate on temporary views that have been cached already. Consider to cache the view <viewName>."
         ]
       },
+      "ANALYZE_UNSUPPORTED_COLUMN_TYPE" : {
+        "message": [
+          "The ANALYZE TABLE FOR COLUMNS command does not support on <colName> (data type: <dataType>) in table <tableName>."
+        ]
```

Comment:
The error message is more reasonable as follows ?
The ANALYZE TABLE FOR COLUMNS command can't operate on <colName> in table <tableName>, because it don't support <dataType>.

Reply:
How about:
```
The ANALYZE TABLE FOR COLUMNS command does not support the type <columnType> of the column <colName> in the table <tableName>.
```